packages/mobile/src/fiatExchanges/reducer.ts

```diff
@@ -5,14 +5,52 @@ import i18n from 'src/i18n'
 import { getRehydratePayload, REHYDRATE } from 'src/redux/persist-helper'
 import { RootState } from 'src/redux/reducers'
 
+export enum CiCoProvider {
+  MOONPAY = 'MOONPAY',
+  RAMP = 'RAMP',
+  SIMPLEX = 'SIMPLEX',
+  TRANSAK = 'TRANSAK',
+}
+
+export const providersDisplayInfo: { [provider in CiCoProvider]: ProviderFeedInfo } = {
+  [CiCoProvider.MOONPAY]: {
+    name: 'Moonpay',
+    icon:
+      'https://firebasestorage.googleapis.com/v0/b/celo-mobile-mainnet.appspot.com/o/images%2Fmoonpay.png?alt=media&token=3617af49-7762-414d-a4d0-df05fbc49b97',
+  },
+  [CiCoProvider.RAMP]: {
+    name: 'Ramp',
+    icon:
+      'https://firebasestorage.googleapis.com/v0/b/celo-mobile-mainnet.appspot.com/o/images%2Framp.png?alt=media&token=548ab5b9-7b03-49a2-a196-198f45958852',
+  },
+  [CiCoProvider.SIMPLEX]: {
+    name: 'Simplex',
+    icon:
+      'https://firebasestorage.googleapis.com/v0/b/celo-mobile-mainnet.appspot.com/o/images%2Fsimplex.jpg?alt=media&token=6037b2f9-9d76-4076-b29e-b7e0de0b3f34',
+  },
+  [CiCoProvider.TRANSAK]: {
+    name: 'Transak',
+    icon:
+      'https://storage.cloud.google.com/celo-mobile-mainnet.appspot.com/images/transak-icon.png',
+  },
+}
+
+export interface TxHashToProvider {
+  [txHash: string]: CiCoProvider | undefined
+}
+
 interface ProviderFeedInfo {
   name: string
   icon: string
 }
 
+interface TxHashToDisplayInfo {
+  [txHash: string]: ProviderFeedInfo | undefined
+}
+
 export interface State {
-  lastUsedProvider: ProviderFeedInfo | null
-  txHashToProvider: { [txHash: string]: ProviderFeedInfo | undefined }
+  lastUsedProvider: CiCoProvider | null
```

Comment:
I haven't seen a migration for this. Do we need one?

Reply:
Yeah, good point. I think it's fine to lose the info, but we should make sure it's set to null and doesn't have info formatted in an unexpected way.